deployability/modules/testing/tests/helpers/agent.py

```diff
@@ -378,40 +471,23 @@ def get_agents_information(wazuh_api: WazuhAPI) -> list:
     def get_agent_status(wazuh_api: WazuhAPI, agent_name) -> str:
```

Comment:
Decorate this method as `@staticmethod`

Reply:
This method depends on the WazuhApi instantiation so the decorator could alter its operation

Reply:
Decorating the function with `@staticmethod` shouldn't alter its operation. We agreed to test this function without changes and defer the change request to the integration with the Central Components branch, which we will merge next to this PR.